e2e/atlas/private_endpoint_test.go

```diff
@@ -315,12 +315,71 @@ func TestPrivateEndpointsAzure(t *testing.T) {
 	})
 }
 
+var regionsGCP = []string{
+	"CENTRAL_US",
+	"EASTERN_US",
+	"US_EAST_4",
+	"NORTH_AMERICA_NORTHEAST_1",
+	"NORTH_AMERICA_NORTHEAST_2",
+	"SOUTH_AMERICA_EAST_1",
+	"WESTERN_US",
+	"US_WEST_2",
+	"US-WEST_3",
+	"US-WEST_4",
+	"EASTERN_ASIA_PACIFIC",
+	"ASIA_EAST_2",
+	"NORTHEASTERN_ASIA_PACIFIC",
+	"ASIA_NORTHEAST_2",
+	"ASIA_NORTHEAST_3",
+	"SOUTHEASTERN_ASIA_PACIFIC",
+	"ASIA_SOUTH_1",
+	"ASIA_SOUTH_2",
+	"AUSTRALIA_SOUTHEAST_1",
+	"AUSTRALIA_SOUTHEAST_2",
+	"ASIA_SOUTHEAST_2",
+	"WESTERN_EUROPE",
+	"EUROPE_NORTH_1",
+	"EUROPE_WEST_2",
+	"EUROPE_WEST_3",
+	"EUROPE_WEST_4",
+	"EUROPE_WEST_6",
+	"EUROPE_CENTRAL_2",
+}
+
```

Comment:
I think we agreed to add the e2e test only once you have added the delete command.
Keep in mind that not all the GPC regions are available on cloud-dev, you should test them on the UI before adding the region here.

Reply:
nice to know thanks for the heads up. just tested and will be removing 2 regions here.